src/main/java/recipeio/ui/UI.java

```diff
@@ -72,58 +72,60 @@ public static void printAddMessage(Recipe recipe, int recipeListSize) {
-        System.out.println("\tOkay chef! I have deleted this recipe from your recipe book:");
-        System.out.print("\t\t");
+        System.out.println("Okay chef! I have deleted this recipe from your recipe book:");
+        System.out.print("\t");
         System.out.print(recipe.toString());
-        System.out.println("\n\tYou now have " + recipeListSize + " recipes in your recipe book. Keep adding some!");
+        System.out.println("\nYou now have " + recipeListSize + " recipes in your recipe book. Keep adding some!");
     }
 
     /**
      * Prints the valid meal categories
      */
     public static void printValidMealCategories() {
-        System.out.println("\tAccepted meal categories are:");
-        System.out.println("\t\tBreakfast");
-        System.out.println("\t\tLunch");
-        System.out.println("\t\tDinner");
-        System.out.println("\t\tDessert");
-        System.out.println("\t\tAppetizer");
-        System.out.println("\t\tGeneral");
+        System.out.println("Accepted meal categories are:");
+        System.out.println("\tBreakfast");
+        System.out.println("\tLunch");
+        System.out.println("\tDinner");
+        System.out.println("\tDessert");
+        System.out.println("\tAppetizer");
+        System.out.println("\tGeneral");
     }
 
     /**
      * Prints list of accepted instructions.
      */
     public static void printInstructions(){
-        System.out.println("\tAccepted commands are:");
-        System.out.println("\thelp: shows available commands\n");
-        System.out.println("\tlist: shows you list of recipes");
-        System.out.println("\t\tInput Example: list\n");
-        System.out.println("\tadd NAME/MINUTES/KCALS/ALLERGIES/CATEGORY/URL: adds a recipe");
-        System.out.println("\t\tInput Example: add pizza/34/340/eggs/dinner/www.food.com\n");
-        System.out.println("\tdelete LIST_NUMBER: deletes a recipe at a given index");
-        System.out.println("\t\tInput Example: delete 1\n");
-        System.out.println("\tfind kw KEYWORD: finds recipes with a given keyword");
-        System.out.println("\t\tInput Example: find kw pizza\n");
-        System.out.println("\tfind date YYYY-MM-DD: finds recipes added on a given date");
-        System.out.println("\t\tInput Example: find date 2024-03-28\n");
-        System.out.println("\tfilter ALLERGY: lists the recipes that do not contain this allergen");
-        System.out.println("\t\tInput Example: filter dairy\n");
-        System.out.println("\texit: to leave the program");
+        System.out.println("Accepted commands are:");
+        System.out.println("help: shows available commands\n");
+        System.out.println("list: shows you list of recipes");
+        System.out.println("\tInput Example: list\n");
+        System.out.println("add NAME, MINUTES, KCALS, SINGULAR TENSE, SPACE SEPARATED ALLERGIES, CATEGORY, " +
+                "URL: adds a recipe");
```

Comment:
Please follow the format of console output. It is "command:", then explanation.

You use comma to separate the parameters, but also in `SINGULAR TENSE, SPACE SEPARATED ALLERGIES`, when this is actually one parameter. 

You could remove the comma in the middle.